src/utils/verse.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
the utils is now more > 150 LoC. but not sure how we want to split the files. Open for a discussion

Reply:
I think we can put it inside `utils/sampleVerse.ts`

Reply:
Can we remove this line?